### PR TITLE
docs(contributing): add link check information

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
title

  

## PR Context  

@TravisEz13 told me to do this
Travis if your reading this just know I have to update the other file please no merge yet

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [X] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Please see [Building PowerShell](../README.md#building-the-repository).
 
 #### Testing PowerShell
 
-Please see PowerShell [Testing Guidelines - Running Tests Outside of CI][running-tests-outside-of-ci] on how to test you build locally.
+Please see PowerShell [Testing Guidelines - Running Tests Outside of CI][running-tests-outside-of-ci] on how to test you build locally.
```
Comment (adityapatwardhan) [inline .github/CONTRIBUTING.md]: Please remove extra space at the end of line.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ Additional references:
 * After submitting your pull request,
   our [CI system (Azure DevOps Pipelines)][ci-system]
   will run a suite of tests and automatically update the status of the pull request.
-* Our CI contains automated spellchecking. If there is any false-positive,
+* Our CI contains automated spellchecking and link checking (markdown files only for link checking). If there is any false-positive,
```
Comment (adityapatwardhan) [inline .github/CONTRIBUTING.md]:
```suggestion
* Our CI contains automated spellchecking and link checking for markdown files. If there is any false-positive,
```

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by RDIL** [comment]
@adityapatwardhan feedback addressed, @TravisEz13 ready for review.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -60,7 +60,19 @@ To run the spellchecker, follow the steps as follows:
 * install [markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck) by
   `npm install -g markdown-spellcheck` (v0.11.0 or up)
 * run `mdspell "**/*.md" --ignore-numbers --ignore-acronyms --en-us`
-* if the `.spelling` file is updated, commit and push it
+* if the `.spelling` file is updated, commit and push it  
+
+#### Checking links in documentation  
+Documentation is link-checked. We make use of the
+markdown-link-check tool command line tool,
```
[the remark this answers is not in the document]
Comment (RDIL) [reply .github/CONTRIBUTING.md]: I'm well aware, I added the check 😂

[the remark this answers is not in the document]
Comment (alexandair) [reply .github/CONTRIBUTING.md]: I don't follow you. My comment was offering a rewrite of your statement.

[the remark this answers is not in the document]
Comment (alexandair) [reply .github/CONTRIBUTING.md]: Fix a typo: "command line tool" > "command-line tool"

**Comment by RDIL** [comment]
@adityapatwardhan please re-review since requested changes were addressed.

**Review by adityapatwardhan — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -177,7 +189,7 @@ Additional references:
   Company = "Microsoft Corporation"
   Copyright = "Copyright (c) Microsoft Corporation. All rights reserved."
   ```
-
+  is at the top.
```
Comment (TravisEz13) [inline .github/CONTRIBUTING.md]: Please make sure that there is a newline before and after any header (the spelling task will fail if there is not)

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -49,18 +49,30 @@ Please see the [Contributor Guide in `PowerShell/PowerShell-Docs`](https://githu
...
 
 * install [Node.js](https://nodejs.org/en/) (v6.4.0 or up)
 * install [markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck) by
   `npm install -g markdown-spellcheck` (v0.11.0 or up)
 * run `mdspell "**/*.md" --ignore-numbers --ignore-acronyms --en-us`
-* if the `.spelling` file is updated, commit and push it
+* if the `.spelling` file is updated, commit and push it  
+
+#### Checking links in documentation  
+Documentation is link-checked. We make use of the
```
Comment (TravisEz13) [inline .github/CONTRIBUTING.md]: I would suggest saying that this and markdown syntax checking (like the newline before and after headers.) can be checked by running `start-pspester -includecommontests`

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -49,18 +49,30 @@ Please see the [Contributor Guide in `PowerShell/PowerShell-Docs`](https://githu
...
+To run the spellchecker, follow these steps:
 
 * install [Node.js](https://nodejs.org/en/) (v6.4.0 or up)
 * install [markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck) by
   `npm install -g markdown-spellcheck` (v0.11.0 or up)
 * run `mdspell "**/*.md" --ignore-numbers --ignore-acronyms --en-us`
-* if the `.spelling` file is updated, commit and push it
+* if the `.spelling` file is updated, commit and push it  
+
+#### Checking links in documentation
```
Comment (TravisEz13) [inline .github/CONTRIBUTING.md]: missing newline after header

Comment (RDIL) [reply .github/CONTRIBUTING.md]: oops

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -49,18 +49,30 @@ Please see the [Contributor Guide in `PowerShell/PowerShell-Docs`](https://githu
 
 #### Spellchecking documentation
 
-Documentation are spellchecked. We make use of the
+Documentation is spellchecked. We use the
```
Comment (TravisEz13) [inline .github/CONTRIBUTING.md]: This file apparently isn't being checked for markdown syntax issues.  
You can add it by updating this list https://github.com/PowerShell/PowerShell/blob/03cd3a2548a8029dc44b7c1155c992ea9d5f960b/test/common/markdown/markdown.tests.ps1#L57-L67

Not a requirement.  If you are using, vscode, you can just install the markdownlinq extension and you should get the issues as well.

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (RDIL) [reply .github/CONTRIBUTING.md]: 👍

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (RDIL) [reply .github/CONTRIBUTING.md]: Okay 👍

**Comment by TravisEz13** [comment]
Thanks a lot.

**Pull request merged**